src/main/resources/org/biouno/unochoice/common/checkboxContent.jelly

```diff
@@ -1,10 +1,54 @@
 <?jelly escape-by-default='true' ?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
   xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
-  xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
+  xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project" xmlns:transitionWrapper="transitionWrapper">
+    <d:taglib uri="transitionWrapper">
+      <d:tag name="randomTableIdBlock">
+          <j:choose>
+              <j:when test="${divBasedFormLayout}">

```

Comment:
Was wondering where that `divBasedFormLayout` was being defined. Answer: Jenkins sets in the page context if the version uses divs, and not tables.

https://github.com/jenkinsci/jenkins/blob/bdb5dce566566e8724679564fe612130c064580f/core/src/main/java/hudson/Functions.java#L305